homeassistant/components/nordpool/coordinator.py

```diff
@@ -44,9 +44,10 @@ def __init__(self, hass: HomeAssistant, config_entry: NordPoolConfigEntry) -> No
             name=DOMAIN,
         )
         self.client = NordPoolClient(session=async_get_clientsession(hass))
-        self.unsub: Callable[[], None] | None = None
+        self.data_unsub: Callable[[], None] | None = None
+        self.listener_unsub: Callable[[], None] | None = None
 
-    def get_next_interval(self, now: datetime) -> datetime:
+    def get_next_data_interval(self, now: datetime) -> datetime:
```

Comment:
Why is `now` needed? It's not in use. And `dt_util.utcnow()` is used below, which is also the input when used. Same below on line 63